src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -318,10 +318,15 @@ private void Poll()
                     var pauseThese = currentAssignment.Except(resumeThese).ToList();
                     PausePartitions(pauseThese);
                     ResumePartitions(resumeThese);
-                    
+
                     ProcessResult(partitionsToFetch, _consumer.Consume(_settings.PollTimeout));
                 }
             }
+            // Workaroud for https://github.com/confluentinc/confluent-kafka-dotnet/issues/1366
+            catch (ConsumeException ex) when (ex.Message.Contains("Broker: Unknown topic or partition") && _settings.AutoCreateTopicsEnabled)
```

Comment:
Since there is some issue with how `AutoCreateTopicsEnabled` works on librdkafka library level, and even when it is enabled sometimes topics may not be created at the moment of assignment - we will skip "unknown topic" error, since it is recommended by confluent driver developers: https://github.com/confluentinc/confluent-kafka-dotnet/issues/1366#issuecomment-666690755

Reply:
I do not like using "magic strings" very much, but seems like this is not some constant value from confluent driver.